saleor/product/tasks.py

```diff
@@ -29,3 +35,27 @@ def update_variants_names(product_type_pk, saved_attributes_ids):
     instance = ProductType.objects.get(pk=product_type_pk)
     saved_attributes = Attribute.objects.filter(pk__in=saved_attributes_ids)
     return _update_variants_names(instance, saved_attributes)
+
+
+@app.task
+def update_product_minimal_variant_price_task(product_pk):
+    product = Product.objects.get(pk=product_pk)
+    update_product_minimal_variant_price(product)
+
+
+@app.task
+def update_products_minimal_variant_prices_task(product_pks):
+    products = Product.objects.filter(pk__in=product_pks)
+    update_products_minimal_variant_prices(products)
+
+
+@app.task
+def update_all_products_minimal_variant_prices_task():
```

Comment:
Is this task used?

Reply:
No. I've created this one with the intend to plug it in the future to e.g. celery-beat with a periodic task.

With the introduction of a `Datetime` as the `Sale's` _start_ and _end_ dates we would need something more granular, ideally something that would update the `minimal variant prices` at the exact time the sale's start/end.